src/app/view/components/login/login-component/login-container.component.ts

```diff
@@ -28,6 +29,7 @@ export class LoginContainerComponent {
 
     constructor(
         private dotRouterService: DotRouterService,
+        private dotNavigationService: DotNavigationService,
```

Comment:
The include of this service here is needed to bind the events on User/Menu changes.

Reply:
Include this in this component just because we need to instace it before we use it doesn't seems right to me... If I'm doing a clean up unused code I'll certainly remove this because in a few months I will forgot why it was there.

We need to come up with a better solution for it. @freddyucv ideas?